practice-two/src/pages/Analytics/Analytics.jsx

```diff
@@ -158,7 +186,7 @@ const Analytics = () => {
           // Show message when list is empty
           <p className='empty__message'>{MESSAGES.EMPTY_LIST}</p>
         )}
-        {isError && showToastInfo(MESSAGES.GET.ERRORS.API_FAILED)}
+        {isError && showToastInfo(MESSAGES.API_FAIL)}
```

Comment:
what is API fail?

Reply:
Because when server has error, we can't get API from server.
I let the message to inform that server has error so can't get API 
I change it to a verb to make it clearly

 
![image](https://github.com/datdragon777/React-Training/assets/74806826/75f4f35e-b4f3-46c2-80e4-d735c1659ac4)